index.js

```diff
@@ -0,0 +1,166 @@
+// import all necessary modules form node or locally
+const chalk = require('chalk');
+const readline = require('readline-sync');
+const randomcolor = require('randomcolor');
+
+// Object with accepted inputs
+const allowedInput = {
+  color: [
+    'red',
+    'orange',
+    'yellow',
+    'green',
+    'blue',
+    'purple',
+    'pink',
+    'monochrome',
+  ],
+  hue: ['light', 'dark'],
+};
+
+//initialize a random hex color variable and print it out
+let colorChoice = randomcolor();
+const symbol = '#';
+
+// get the user input from the process.argv if supplied//const Args = process.argv[2].split('x');
+// creates an array from third argument onwards
+const Args = process.argv.slice(2);
+
+//initialize height and width
+let measures;
+const minHeight = 5;
+const minWidth = 19;
```

Comment:
Since these three variables are only required in a certain case (if the user has passed in the WWxHH), I would put them inside the condition block where that is true.